doc/installationguide.md

```diff
@@ -21,21 +21,51 @@ These are the parameters that can be configured in the global section:
         port: 4041
     	}
 ```
-* **stats**: configure the periodic collection of statistics. Use `interval` in miliseconds to set the time between stats writings.
+* **stats**: configure the periodic collection of statistics. Use `interval` in milliseconds to set the time between stats writings.
 ```
     stats: {
         interval: 100
     }
 ```
-* **authentication**: authentication data, for use in retrieving tokens for devices with a trust token (just needed in scenarios with security enabled in the Context Broker side). E.g.:
-```
-	{
-        host: 'localhost',
-        port: '5000',
-        user: 'iotagent',
-        password: 'iotagent'
-	}
-```
+* **authentication**: authentication data, for use in retrieving tokens for devices with a trust token (required in scenarios with security enabled in the Context Broker side). 
```

Comment:
`enabled` is not include in this documentation

Reply:
fix by 9ea2773